misc/roundtrip

```diff
@@ -22,6 +22,22 @@ export LC_ALL=C
 
 READTAGS=${1:-./readtags}
 UNITS=${2:-./Units}
+shift 2
+
+MINITRIP=
+while [ $# -gt 0 ]; do
+    case $1 in
+	--minitrip)
```

Comment:
The option is implemented here.

Reply:
Oh, I see.

I did not understand that e6409f9 was one of commits in this PR.  I went to e6409f9 directly last night.
I checked https://github.com/universal-ctags/ctags/commits/master/misc/roundtrip.

`minitrip` is really an internal option.  It is not an important information for readers.

> *minitrip* internal option was introduced because roundtrip target
> took too long time on CI/CD environments. The option is automatically
> enabled when running the target on the environments.
> 
> When the option is enabled, the target uses only marked `expected.tags`
> files instead of using all the `expected.tags` files. An empty file

This could be;

> On CI/CD environments, the target uses only marked `expected.tags`
> files instead of using all the `expected.tags` files because the roundtrip target
> took too long time. An empty file ...

How about this?

Reply:
I agree with you. Thank you. I will update the commit.